src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(ColorSupplier.getRandomColor());
+        ball.setNumber(new Random().nextInt(100));
```

Comment:
Consider creating a single instance of Random and reuse it, instead of creating a new Random instance each time you call getRandomBall().

Also magic numbers should be class constants with proper naming